src/agent.c

```diff
@@ -1633,9 +1748,12 @@ int agent_process_turn_allocate(juice_agent_t *agent, const stun_message_t *msg,
 				JLOG_INFO("Got STUN mapped address %s from TURN server", mapped_str);
 			}
 
-			if (agent_add_local_reflexive_candidate(agent, ICE_CANDIDATE_TYPE_SERVER_REFLEXIVE,
-			                                        &msg->mapped)) {
-				JLOG_WARN("Failed to add local peer reflexive candidate from TURN mapped address");
+			if (entry->transport != JUICE_TRANSPORT_TCP) {
```

Comment:
I think that if the entry uses a secondary UDP socket for some reason, the reflexive candidate is useless anyway.